packages/amazonq/.changes/next-release/Feature-5576420e-6db2-4045-a99f-afced496da0f.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
```

Comment:
should be `fix` instead?

Reply:
No this is not a bug

Reply:
but i feel CX will expect something to come out if we mark it as a Feature

Reply:
it's just my insight, not a requirement